sdks/java/extensions/sql/datacatalog/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/datacatalog/DataCatalogBigQueryIT.java

```diff
@@ -69,9 +73,18 @@
 
     @Test
     public void testRead() throws Exception {
-      bigQuery.insertRows(ID_NAME_SCHEMA, row(1, "name1"), row(2, "name2"), row(3, "name3"));
-
       TableReference bqTable = bigQuery.tableReference();
+
+      // Streaming inserts do not work with DIRECT_READ mode, there is a several hour lag.
+      PCollection<Row> data =
+          writePipeline.apply(Create.of(row(1, "name1"), row(2, "name2"), row(3, "name3")));
+      data.apply(
+          BigQueryIO.<Row>write()
+              .withSchema(BigQueryUtils.toTableSchema(ID_NAME_SCHEMA))
+              .withFormatFunction(BigQueryUtils.toTableRow())
```

Comment:
I would suggest to add `.withMethod(Method.FILE_LOADS)` to be explicit. It connects it better with the comment above. And is protected in case the default changes.

Reply:
Done